src/main/java/seedu/address/model/inventorymodel/InventoryModelManager.java

```diff
@@ -126,19 +122,19 @@ public void setItem(Item target, Item editedItem) {
         inventoryBook.setItem(target, editedItem);
     }
 
-    //=========== Filtered Item List Accessors =============================================================
+    //=========== Item List Accessors =============================================================
 
     /**
      * Returns an unmodifiable view of the list of {@code Item} backed by the internal list of
      * {@code versionedInventoryBook}
      */
     @Override
-    public ObservableList<Item> getFilteredItemList() {
-        return filteredItems;
+    public ObservableList<Item> getFilteredAndSortedItemList() {
+        return new SortedList<>(filteredItems, ITEM_COMPARATOR);
     }
 
     @Override
-    public void updateFilteredItemList(Predicate<Item> predicate) {
+    public void updateFilteredAndSortedItemList(Predicate<Item> predicate) {
         requireNonNull(predicate);
         filteredItems.setPredicate(predicate);
     }
```

Comment:
I don't think the "Sorted" part is accurate here. The old name `updateFilteredItemList` sounds more appropriate

Reply:
I changed it to updateItemListFilter!